roles/pulp_common/vars/Debian.yml

```diff
@@ -7,6 +7,7 @@ pulp_preq_packages:
   - gcc              # For psycopg2
   - git              # For source install
   - sudo
+  - gpg              # For the signing service via the gpg python module
```

Comment:
Shouldn't this be `gnupg` ?

Reply:
`gpg` is just the binary. 

"If you want full capabilities
(including secret key operations, network access, etc), please
install the "gnupg" package, which pulls in the full suite of tools."

https://debian.pkgs.org/10/debian-main-amd64/gpg_2.2.12-1+deb10u1_amd64.deb.html

We'll see how the tests go, but let me know if you think `gpg` is sufficient for every possible usage.

Reply:
It's passing.

If someone else approves, I will merge & follow up with you later about whether it will be sufficient.